grpc/service_config/service_config.proto

```diff
@@ -352,23 +450,33 @@ message LoadBalancingConfig {
     // balancing policy.
     GrpcLbConfig grpclb = 3;
 
-    PriorityLoadBalancingPolicyConfig priority = 9;
-    WeightedTargetLoadBalancingPolicyConfig weighted_target = 10;
+    // REMAINING POLICIES ARE EXPERIMENTAL -- DO NOT USE
+
+    PriorityLoadBalancingPolicyConfig priority_experimental = 9
+        [json_name = "priority_experimental"];
+    WeightedTargetLoadBalancingPolicyConfig weighted_target_experimental = 10
+        [json_name = "weighted_target_experimental"];
 
-    // EXPERIMENTAL -- DO NOT USE
     // xDS-based load balancing.
-    // The policy is known as xds_experimental while it is under development.
-    // It will be renamed to xds once it is ready for public use.
-    CdsConfig cds = 6;
-    EdsLoadBalancingPolicyConfig eds = 7;
-    LrsLoadBalancingPolicyConfig lrs = 8;
+    CdsConfig cds_experimental = 6 [json_name = "cds_experimental"];
+    XdsClusterResolverLoadBalancingPolicyConfig
+        xds_cluster_resolver_experimental = 11
+        [json_name = "xds_cluster_resolver_experimental"];
+    XdsClusterImplLoadBalancingPolicyConfig xds_cluster_impl_experimental = 12
+        [json_name = "xds_cluster_impl_experimental"];
+    EdsLoadBalancingPolicyConfig eds_experimental = 7
+        [json_name = "eds_experimental"];
+    RingHashLoadBalancingConfig ring_hash_experimental = 13
+        [json_name = "ring_hash_experimental"];
+
+    // Deprecated xDS-related policies.
+    LrsLoadBalancingPolicyConfig lrs_experimental = 8
```

Comment:
(And this one)

Reply:
I intentionally put the deprecated policies in a separate section, so that someone reading the file for the first time doesn't need to worry about these fields.